apps/meteor/app/lib/server/lib/passwordPolicy.ts

```diff
@@ -0,0 +1,44 @@
+import PasswordPolicy from '@rocket.chat/account-utils';
+
+import { settings } from '../../../settings/server';
+
+export const passwordPolicy = new PasswordPolicy.PasswordPolicy({
```

Comment:
Can we import the class only? e.g. (`import { PasswordPolicy } from '@rocket.chat/account-utils'`);

Reply:
Hi! Good idea!